pug/pages/blog-details.pug

```diff
@@ -0,0 +1,77 @@
+                    img(src="../assets/images/blog_image_3.png", alt="Thế hệ máy khoan cắt cnc đầu tiên đã được tái sử dụng")
+                    p But I must explain to you how all this mistaken idea of denouncing pleasure and praising pain was born and I will give you a complete account of the system, and expound the actual teachings of the great explorer of the truth, the master-builder of human happiness. No one rejects, dislikes, or avoids pleasure itself, because it is pleasure, but because those who do not know how to pursue pleasure rationally encounter consequences that are extremely painful. Nor again is there anyone who loves or pursues or desires to obtain pain of itself, because it is pain, but because occasionally circumstances occur in which toil and pain can procure him some great pleasure.
+                    p "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga. Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Temporibus autem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae. Itaque earum rerum hic tenetur a sapiente delectus, ut aut reiciendis voluptatibus maiores alias consequatur aut perferendis doloribus asperiores repellat."
+                    p Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt
+                footer.blog__footer
+                    section.blog__sharing
+                        div.blog-tags
+                            nav
+                                h3.title Tags:    
+                                ul 
+                                    li     
+                                    each tag, index in tagsList
+                                        li
+                                            if index < tagsList.length - 1
+                                                a(href="#") #{tag},
+                                            else
+                                                a(href="#") #{tag}...
```

Comment:
Lặp qua từng tagsList rồi in ra các thẻ `<li>` bên trong `<ul>` chứ, sao lại lồng các thẻ `<li>` bên trong `<li>` là sao

Reply:
Em chuyển thẻ span thành h3 quên xóa anh ạ